meshkeyserverspec.md

```
@@ -0,0 +1,210 @@
+
+The mesh bundle is a *view* of completed responses, CBOR-serialized and
+signed. Cache it rather than regenerate per request:
+
+```sql
+ALTER TABLE responses ADD COLUMN mesh_bundle_cbor BLOB;   -- CBOR(payload)
+ALTER TABLE responses ADD COLUMN mesh_bundle_sig  BLOB;   -- ed25519 over the CBOR
+ALTER TABLE responses ADD COLUMN author_fpr       TEXT;   -- references a fingerprint, not a user
+```
+
+`mesh_bundle_cbor` payload shape (CBOR map):
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)